share/commands/agent-check

```diff
@@ -60,24 +64,11 @@ init_commands
 MYUID=`id | cut -d\( -f2 | cut -d\) -f1`
 if [ "${MYUID}" != 'root' ];then echo "You must be root"; exit; fi
 
-# Get the value of rudder-agent run interval from file /var/rudder/cfengine-community/inputs/run_interval
-if [ -f "${CFE_DIR}/inputs/run_interval" ]; then
-  RUN_INTERVAL=`cat "${CFE_DIR}/inputs/run_interval"`
-  # If the value is not a number, reset to 5
-  if ! test "${RUN_INTERVAL}" -gt 0 2>/dev/null
```

Comment:
shouldn't we move this default value to common.sh to handle missing rudder.json file (i.e. before bootstraping fir example)